### PR TITLE
Updated user guide for list command

### PR DESCRIPTION
**Pull request by shaowi**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -20,8 +20,9 @@ LE TRACKER is a gamified tracking application that allows fast typist to easily
 3. Press Enter to execute it. e.g. typing help and pressing Enter will open the help window.
 
 - List
-  - `list /module`: Lists the names of all the recorded modules
-  - `list /module /{module_code}`: Lists all the videos recorded in that module code
+  - `list`: Lists the names of all the recorded modules
+  - `list /mod /{module_code}`: Lists all the lectures in the specified module
+  - `list /mod /{module_code} /lec {lecture_name}`: Lists all the videos in the specified module and lecture
```
Comment (joytqt-1202) [inline docs/UserGuide.md:25]: Is the / before {module_code} supposed to be there?

**Review by joytqt-1202 — CHANGES_REQUESTED** [review]
Do take a look at lines 24 and 25! Thanks!

File: docs/UserGuide.md (hunk 1 above)
Comment (shaowi) [reply docs/UserGuide.md]: Nope thanks for pointing out!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -18,14 +18,16 @@ LE TRACKER is a gamified tracking application that allows fast typist to easily
...
   - `nav`: Navigates to the top-most context (root)
   - `nav {module_code / lecture_name}`: Navigates relative to the current context to a module or lecture context
   - `nav /mod {module_code / lecture_name} [/lec {lecture_name}]`: Navigates directly to the specified module or lecture context
   - `navb`: Navigates to the parent context of the current context
 - List
-  - `list /module`: Lists the names of all the recorded modules
-  - `list /module /{module_code}`: Lists all the videos recorded in that module code
+  - `list`: Lists the names of all the recorded modules
+  - `list /mod {module_code}`: Lists all the lectures in the specified module
+  - `list /mod {module_code} /lec {lecture_name}`: Lists all the videos in the specified module and lecture
```
Comment (hingen) [inline docs/UserGuide.md:30]: `/mod {module_code}` and `/lec {lecture_name}` should be specified as optional arguments as they are auto injected by the context sensitive system if not specified. Can refer to `add` command user guide documentation.

**Review by hingen — APPROVED** [review]
LGTM

**Pull request merged**